tracetool/modules/media.py

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright (C) Intel Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+
+import re
+from util import *
+
+class traceHandler:
+    phaseMap = {1:'B', 2:'E'}
+
+    def __init__(self, core):
+        self.sharedCtx = core.getContext()
+
+        self.DataDump = {}
+
+        core.regParser(b'ETMI', self.mediaHeaderParse)
```

Comment:
> core.regParser(b'ETMI', self.mediaHeaderParse)

what is this and why it's duplicated in media.py? should that be so?

Reply:
this is media trace header parser. each umd trace need its own header parser, like libva.
it is required because all umd trace share the same ftrace entry, so need a header parser for each trace system, b'ETMI' is short for IntelMediaTraceEvent in little endian, each media trace event raw data start with this id. 
currently have media and libva id and header parser. 
regHandler is for register media trace handler, that is trace level handler.